data/mission.yaml

```
@@ -124,3 +124,24 @@ Escort Wendy To Cocorico Village:
     dialog: Wendy Escort Complete
     payment: 125
     exp addition: 25
+
+Escort Gerud Beer Covet:
+  name: "Escort Gerud Beer Covet"
+  description: >
+    Go to $stopovers, to meet up with the Gerud Beer covet, and the escort them through the Badlands and the
+    Gerud Desert to the Los Gerudos Warehouse ($destination). They'll pay you $payment gold.
+    P.-S.: make sure to not encounter any South Pillager camp on your way.
```

```suggestion
    P.S. Make sure to not encounter any South Pillager camp on your way.
```
I think this might be the normal way?

---

Indeed it is.